setup.py

```diff
@@ -778,25 +782,50 @@ def run(self):
 
         def check_linter_exists(linter):
             if shutil.which(linter) is None:
-                msg = "Please install '%s' in your environment. (hint: 'pip install %s')"
+                msg = "Please install '%s' in your environment. (hint: 'python3 -m pip install %s')"
                 warnings.warn(msg % (linter, linter))
                 sys.exit(1)
 
-        c_files_unfiltered = glob.glob("src_c/**/*.[ch]")
-        c_file_disallow = ["_sdl2", "pypm", "SDL_gfx", "sse2neon.h", "src_c/doc/"]
+        c_files_unfiltered = glob.glob("src_c/*.[ch]")
+        c_files_unfiltered += glob.glob("src_c/**/*.[ch]")
```

Comment:
There two lines should become
`c_files_unfiltered = glob.glob("src_c/**/*.[ch]", recursive=True)`
That way, not only it handles the base of the directory in the same function call, it also actually recurses into nested sub-directories according to the [docs](https://docs.python.org/3/library/glob.html#glob.glob) of the function

Reply:
This is done, thanks.